r/R/dataset-write.R

```diff
@@ -111,6 +121,10 @@ write_dataset <- function(dataset,
                           hive_style = TRUE,
                           existing_data_behavior = c("overwrite", "error", "delete_matching"),
                           max_partitions = 1024L,
+                          max_open_files = 900L,
```

Comment:
Is `900L` here a default in C++? I'm curious where that specific number came from... (it's close to `1024L` or `1000L`, but curious why it's lower?)

Reply:
It is mentioned specifically in the c++ docs. https://github.com/apache/arrow/blob/c5b757fe607b1e5824053da279a727e35e877e0a/cpp/src/arrow/dataset/file_base.h#L368

It contains a nice description 😊

Reply:
Oh that's very nice! Would you mind copying/pasting that into the docs here? IIRC, we can't (easily) link to the C++ docs since the URLs are something like https://arrow.apache.org/docs/cpp/api/dataset.html#_CPPv4N5arrow7dataset29FileSystemDatasetWriteOptions17basename_templateE and change with each version.

Reply:
Of course I will
I missed this in this PR. We did the same for the Python.